modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/net/jms/Commit.java

```diff
@@ -32,6 +31,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.wso2.carbon.messaging.CarbonMessage;
+import org.wso2.carbon.transport.jms.utils.JMSConstants;
```

Comment:
Why do we need to depend on constants in the jms transport implementation ? can we get rid of it ?